src/Monitor.vala

```diff
@@ -28,7 +52,21 @@ namespace Monitor {
         }
 
         public static int main (string [] args) {
-            var app = new MonitorApp ();
+
+            // add command line options 
+            try {
+                var opt_context = new OptionContext ("");
+                opt_context.set_help_enabled (true);
+                opt_context.add_main_entries (cmd_options, null);
+                opt_context.parse (ref args);
+            } catch (OptionError e) {
+                print ("Error: %s\n", e.message);
+                print ("Run '%s --help' to see a full list of available command line options.\n\n", args[0]);
+                return 0;
+            }
+
+            MonitorApp app = new MonitorApp (start_in_background);
```

Comment:
You can simply use `var` like [the current code](https://github.com/stsdc/monitor/blob/0d3edab102814da638dcc8ca0ca1ef1071d3ae60/src/Monitor.vala#L31).